lib/Importer/ONNX.cpp

```diff
@@ -80,6 +80,21 @@ size_t getPad(ArgumentDictionaryTy &dict) {
   return 0;
 }
 
+std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
+  if (dict.count("pads")) {
+    return getShape(dict["pads"]);
+  } else if (dict.count("auto_pad")) {
```

Comment:
no need for else after a return.
http://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
can you also fix a similar issue in the getPad method above? (line 71)